aiconfig-docs/docs/getting-started.md

```diff
@@ -304,78 +279,130 @@ await aiConfig.runWithDependencies(
 
 1. Go to https://lastmileai.dev.
 2. Go to Workbooks page: https://lastmileai.dev/workbooks
 3. Click dropdown from '+ New Workbook' and select 'Create from AIConfig'
 4. Upload `travel.aiconfig.json`
 
-Try out the workbook playground here: **[NYC Travel Workbook](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9)**
+Try out the AI Workbook for this tutorial: **[NYC Travel Workbook](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9)**
+
+<iframe width="100%" height="444" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/5d901493-bbda-4f8e-93c7-dd9a91bf242e" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
 :::note
 We are working on a local editor that you can run yourself. For now, please use the hosted version on https://lastmileai.dev.
 :::
 
-<video controls><source src="https://s3.amazonaws.com/publicdata.lastmileai.com/workbook_editor_480.mov"/></video>
+## Next Steps
 
-```
-
-```
+We have several other tutorials and examples to help get you started:
 
-### Code for Getting Started
-
-Tutorial code can be found [here](https://github.com/lastmile-ai/aiconfig/tree/ad38040ec3d9f0273e006464e01e02b06f2809e9/cookbook/Getting-Started).
-
-## Additional Guides
-
-There is a lot you can do with `aiconfig`. We have several other tutorials to help get you started:
+**How-to Guides**
 
 - [Create an AIConfig from scratch](/docs/overview/create-an-aiconfig)
 - [Run a prompt](/docs/overview/run-aiconfig)
 - [Pass data into prompts](/docs/overview/parameters)
 - [Prompt chains](/docs/overview/define-prompt-chain)
 - [Callbacks and monitoring](/docs/overview/monitoring-aiconfig)
 
-### Cookbooks
+**Cookbook**
 
+Start with these recipes and access more [here](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks):
+
+- [OpenAI Function Calling](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Function-Calling-OpenAI)
 - [CLI Chatbot](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Wizard-GPT)
 - [RAG with AIConfig](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/RAG-with-AIConfig)
-- [Prompt routing](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Basic-Prompt-Routing)
-- [OpenAI function calling](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Function-Calling-OpenAI)
-- [Chain of thought](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Chain-of-Verification)
+- [Prompt Routing](https://github.com/lastmile-ai/aiconfig/tree/main/cookbooks/Basic-Prompt-Routing)
+
+## Coming Soon
+
+- Local editor to iterate on your AIConfigs
+- Evaluation interfaces: allow AIConfig artifacts to be evaluated with user-defined eval functions
+- OpenAI Assistants API support
+- Multi-modal ModelParsers: GPT4-V support, Whisper, HuggingFace image generation
```

Comment:
We already have this! Here are some examples:

- text-to-speech: https://github.com/lastmile-ai/aiconfig/pull/793
- text-to-image (both hugging face and dall-e): https://github.com/lastmile-ai/aiconfig/pull/288, https://github.com/lastmile-ai/aiconfig/pull/460 
- image-to-text: https://github.com/lastmile-ai/aiconfig/pull/821, 
- automatic speech recognition (audio-to-text): https://github.com/lastmile-ai/aiconfig/pull/780